server/views/pages/induction/prePrisonEducation/qualificationLevel.njk

```diff
@@ -0,0 +1,165 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "induction-qualification-level" %}
+{% set title = "What level of secondary school qualification does " + prisonerSummary.firstName + " " + prisonerSummary.lastName + " want to add?" %}
+{% set pageTitle = title %}
+
+{#
+Data supplied to this template:
+  * prisonerSummary - object with firstName and lastName
+  * backLinkUrl - url of the back link
+  * backLinkAriaText - the aria label for the back link
+  * form - form object containing the following fields:
+    * educationLevel - the prisoner's highest level of education
+    * qualificationLevel? - the level for the qualification being added
+  * errors? - validation errors
+#}
+
+{# Setup page based on education level #}
+{% set options = [
+  {
+    value: "ENTRY_LEVEL",
+    text: "ENTRY_LEVEL" | formatQualificationLevel,
+    hint: {
+      text: "ENTRY_LEVEL" | formatQualificationLevelHint
+    },
+    checked: form.qualificationLevel === "ENTRY_LEVEL"
+  },
```

Comment:
This logic has been copied over from Gary's template, but with our approach for checkboxes (or in this case radio buttons) applied.

Reply:
Yes, that is the correct logic to use 👍